fluentui_listitem/src/main/java/com/microsoft/fluentui/tokenized/tabItem/TabItem.kt

```diff
@@ -274,14 +283,22 @@ fun TabItem(
                 )
                 {
                     Box(
-                        modifier = Modifier
+                        modifier = if(indicatorColor != null) {
+                            Modifier
                             .height(3.dp)
                             .width(indicatorWidth)
-                            .background(shape = RoundedCornerShape(indicatorCornerRadiusSize), color = textColor)
+                            .background(shape = RoundedCornerShape(indicatorCornerRadiusSize), brush = indicatorColor)
```

Comment:
why are we using brush and not color?

Reply:
Brush is itself an Object containing all the information about gradient and colors used. As far as I am aware, it is not possible to use gradient as a color object neither is it possible to cast Brush to Color.